R/IN22.R

```diff
@@ -0,0 +1,139 @@
+
+#################################################
+# INPUTS FOR THIS SCRIPT - CHANGE THIS SECTION
+
+input_folder = "input/SON25/"
+
+input_file = "2025-01-01-in22-full-dataset.csv"
+
+output_folder_prefix = "../son/content"
+domain = "intermediate_outcomes"
+subdomain = "routes_into_work_(16_to_29_years)"
+indicator_name = "entry_to_higher_education"
+version = "3.0"
+
+indicator_code = "IN22"
+
+
+
+############
+# IMPORTS
+
+source("_shared.R")
+
+
+
+###############
+# THE SCRIPT
+
+input_file_path = paste0(input_folder, input_file)
+
+# Open the input file
+data = read.csv(input_file_path)
+
+# Remove some columns
+data = data_frame__remove_columns(
+  data,
+  "variable_used",
+  "age_used",
+  "weight_used",
+  "year_used",
+  "label"
+)
+
+
+##########################################
```

Comment:
In the [R script for IN21](https://github.com/ukgov-equality-hub/smc-son/blob/main/R/IN21.R#L44-L45) we have a couple of lines here to save out the "full dataset" CSV file:

```R
csv_filename = generate_csv_file_name()
save_data_frame(data, csv_filename)
```